vision/google/cloud/vision/face.py

```diff
@@ -353,41 +449,110 @@ class LandmarkTypes(Enum):
     See:
     https://cloud.google.com/vision/reference/rest/v1/images/annotate#Type_1
     """
-    UNKNOWN_LANDMARK = 'UNKNOWN_LANDMARK'
-    LEFT_EYE = 'LEFT_EYE'
-    RIGHT_EYE = 'RIGHT_EYE'
-    LEFT_OF_LEFT_EYEBROW = 'LEFT_OF_LEFT_EYEBROW'
-    RIGHT_OF_LEFT_EYEBROW = 'RIGHT_OF_LEFT_EYEBROW'
-    LEFT_OF_RIGHT_EYEBROW = 'LEFT_OF_RIGHT_EYEBROW'
-    RIGHT_OF_RIGHT_EYEBROW = 'RIGHT_OF_RIGHT_EYEBROW'
-    MIDPOINT_BETWEEN_EYES = 'MIDPOINT_BETWEEN_EYES'
-    NOSE_TIP = 'NOSE_TIP'
-    UPPER_LIP = 'UPPER_LIP'
-    LOWER_LIP = 'LOWER_LIP'
-    MOUTH_LEFT = 'MOUTH_LEFT'
-    MOUTH_RIGHT = 'MOUTH_RIGHT'
-    MOUTH_CENTER = 'MOUTH_CENTER'
-    NOSE_BOTTOM_RIGHT = 'NOSE_BOTTOM_RIGHT'
-    NOSE_BOTTOM_LEFT = 'NOSE_BOTTOM_LEFT'
-    NOSE_BOTTOM_CENTER = 'NOSE_BOTTOM_CENTER'
-    LEFT_EYE_TOP_BOUNDARY = 'LEFT_EYE_TOP_BOUNDARY'
-    LEFT_EYE_RIGHT_CORNER = 'LEFT_EYE_RIGHT_CORNER'
-    LEFT_EYE_BOTTOM_BOUNDARY = 'LEFT_EYE_BOTTOM_BOUNDARY'
-    LEFT_EYE_LEFT_CORNER = 'LEFT_EYE_LEFT_CORNER'
-    RIGHT_EYE_TOP_BOUNDARY = 'RIGHT_EYE_TOP_BOUNDARY'
-    RIGHT_EYE_RIGHT_CORNER = 'RIGHT_EYE_RIGHT_CORNER'
-    RIGHT_EYE_BOTTOM_BOUNDARY = 'RIGHT_EYE_BOTTOM_BOUNDARY'
-    RIGHT_EYE_LEFT_CORNER = 'RIGHT_EYE_LEFT_CORNER'
-    LEFT_EYEBROW_UPPER_MIDPOINT = 'LEFT_EYEBROW_UPPER_MIDPOINT'
-    RIGHT_EYEBROW_UPPER_MIDPOINT = 'RIGHT_EYEBROW_UPPER_MIDPOINT'
-    LEFT_EAR_TRAGION = 'LEFT_EAR_TRAGION'
-    RIGHT_EAR_TRAGION = 'RIGHT_EAR_TRAGION'
-    LEFT_EYE_PUPIL = 'LEFT_EYE_PUPIL'
-    RIGHT_EYE_PUPIL = 'RIGHT_EYE_PUPIL'
-    FOREHEAD_GLABELLA = 'FOREHEAD_GLABELLA'
-    CHIN_GNATHION = 'CHIN_GNATHION'
-    CHIN_LEFT_GONION = 'CHIN_LEFT_GONION'
-    CHIN_RIGHT_GONION = 'CHIN_RIGHT_GONION'
+    # Unknown face landmark detected. Should not be filled.
+    UNKNOWN_LANDMARK = 0
```

Comment:
So I converted the HTTP version to handle it with integers so that both PB and HTTP versions handle the types the same way.

I might be able to change the HTTP version so it just uses the values that are returned but that seems kind of like the wild west.